core/RhythmMusicGeneration/src/main/java/org/jjazz/rhythmmusicgeneration/api/SongSequenceBuilder.java

```diff
@@ -388,10 +397,8 @@ public void makeSequenceExportable(SongSequence songSequence, boolean ignoreMidi
 
         // Add additional song part tempo changes
         int lastTempoFactor = tempoFactor;
-        var spts = songContextWork.getSongParts();
-        for (i = 1; i < spts.size(); i++)
+        for (SongPart spt : spts)
```

Comment:
you should add the marker in this loop: the SongPart tick position is already calculated for you (move the computatino of beatPos and tickPos outside of the TempoFactor specific part.